pkgdown/_pkgdown.yml

```diff
@@ -51,21 +51,21 @@ articles:
 
 reference:
 - title: Build
-  
+
 - subtitle: Build the complete site
   contents:
   - build_site
   - clean_site
   - preview_site
   - pkgdown_sitrep
-  
+
 - subtitle: Build part of a site
   desc: These functions are useful for rapid iteration when you're working on a
     specific part of your site.
   contents:
-  - starts_with("build_")
-  - -build_site
-  - -build_site_github_pages
+  - has_concept("site components")
```

Comment:
The only downside here is that we lose the alphabetical ordering, but I don't think that's a big deal.